installer/roles/image_build/templates/Dockerfile.j2

```diff
@@ -80,30 +61,19 @@ ADD requirements/requirements_ansible.txt \
     requirements/requirements_tower_uninstall.txt \
     requirements/requirements_git.txt \
     /tmp/requirements/
-RUN scl enable rh-postgresql10 'cd /tmp && VENV_BASE="/var/lib/awx/venv" make requirements'
+RUN cd /tmp && VENV_BASE="/var/lib/awx/venv" make requirements_awx requirements_ansible_py3
 
-RUN yum -y remove cyrus-sasl-devel \
+RUN dnf -y remove *-devel \
   gcc \
   gcc-c++ \
-  krb5-devel \
-  libtool-ltdl-devel \
-  libxml2-devel \
-  libxslt-devel \
-  openldap-devel \
-  rh-postgresql10-postgresql-devel \
-  python-devel \
-  python36-devel \
-  nodejs \
-  xmlsec1-devel \
-  xmlsec1-openssl-devel
+  nodejs
 
 # makes postgres available for data migrations when upgrading
 RUN export PATH=/usr/pgsql-10/bin:$PATH
 
-RUN yum -y clean all
+RUN dnf -y clean all
```

Comment:
Applies to the dev env as well, but this perhaps should be moved to after installing kubectl/oc.

Reply:
Will make a separate PR later and do this in both places.